scripts/r.import/r.import.py

```diff
@@ -168,6 +178,7 @@ def main():
     title = options["title"]
     if flags["e"] and not output:
         output = "rimport_tmp"  # will be removed with the entire tmp location
```

Comment:
As @petrasovaa suggested,
```suggestion
        output = gs.append_node_pid("rimport_tmp")  # will be removed with the entire tmp location
```